samza-core/src/main/java/org/apache/samza/storage/blobstore/BlobStoreRestoreManager.java

```diff
@@ -119,13 +119,18 @@ public BlobStoreRestoreManager(TaskModel taskModel, ExecutorService restoreExecu
 
   @Override
   public void init(Checkpoint checkpoint) {
+    init(checkpoint, false);
+  }
+
+  public void init(Checkpoint checkpoint, Boolean getDeletedBlob) {
```

Comment:
This is needed because TaskRestoreManager interface has init(Checkpoint). We only need getDeleted version of it for blob store, so created another version here.